SocialFilter/sections/specific.txt

```diff
@@ -24,6 +24,9 @@ citydog.by##.headerBlockShare
 don24.ru##.socials-and-print
 newser.com###divSocialAndCommentsButtonsTop > a:not([id="aShareComments"])
 fishki.net##.content__figure__share__social
+||vk.com/widget_community.php^$domain=sims4pack.ru,third-party
+sims4pack.ru##.menu-block:has(#vk_groups)
```

Comment:
Please move this and the following rule to css_extended. Also please modify both ext-css rules to include parent element:

.block-right > .menu-block